requirements.txt

```diff
@@ -0,0 +1,13 @@
+certifi==2020.11.8
```

Comment:
This file still needs to be deleted. Sorry the module 2 instructions were out of date and asked you to create it.